test/remittance.js

```diff
@@ -0,0 +1,168 @@
+const truffleAssert = require('truffle-assertions');
+const Remittance = artifacts.require('Remittance');
+
+const { toBN, toWei, fromAscii } = web3.utils;
+
+const zeroAddress = '0x0000000000000000000000000000000000000000';
+
+contract('Remittance', accounts => {
+  const [owner, carol, someoneElse] = accounts;
+  let contract;
+
+  const bobCorrectPassword = fromAscii('bob-super-secure-password');
+  const carolCorrectPassword = fromAscii('carol-super-secure-password');
+
+  const bobWrongPassword = fromAscii('bob-wrong-password');
+  const carolWrongPassword = fromAscii('carol-wrong-password');
+
+  beforeEach(async () => {
+    contract = (await Remittance.new(false, { from: owner, gas: 3000000 })).contract;
+  });
+
+  context('constructor()', () => {
+    it('should deploy the contract correctly', async () => {
+      assert.ok(contract);
+      assert.strictEqual(await contract.methods.getOwner().call(), owner);
+      assert.strictEqual(await contract.methods.isPaused().call(), false);
+    });
+  });
+  
+  
+  context('deposit()', async () => {
+    let hash;
+    beforeEach(async () => {
+      // Arrange
+      hash = await contract.methods.calcuateHash(carol, bobCorrectPassword, carolCorrectPassword).call();
```

Comment:
Seeing the `calcuateHash` typo everywhere is jarring ;)

Reply:
sorry, my bad